wp-post-meta-revisions.php

```diff
@@ -134,6 +134,10 @@ public function _wp_save_revisioned_meta_fields( $revision_id ) {
 		foreach ( $this->_wp_post_revision_meta_keys() as $meta_key ) {
 			$meta_value = get_post_meta( $post_id, $meta_key );
 
+			if ( empty( $meta_value ) ) {
```

Comment:
I think this needs to do `false ===` because it could be: `0`, `string(0)`, `array()`